events/messageReactionAdd.js

```diff
@@ -23,4 +23,27 @@ module.exports = (client, reaction, user) => {
 			}
 		}
 	}
+
+
+	// AnonSuggestion reaction listening
+	const {modBot} = client.config.channelIDs;
+	const {suggestions} = client.config.channelIDs;
+
+	if (reaction.message.channel == client.channels.get(modBot)) {
+		if(user.id != client.user.id) {
+			if(reaction.emoji.name == '✔') {
+				let content = reaction.message.content;
```

Comment:
`let {content} = reaction.message;`

Reply:
Can also be done via the `lastIndexOf` method of string coupled with `substring()`.

```
const suggestionString = reaction.message.content;
const lastIndex = suggestionString.content.lastIndexOf(' ');
const content = suggestionString.substring(0, lastIndex);
```

Reply:
Done these but better now.